CondFormats/L1TObjects/interface/CaloParams.h

```diff
@@ -38,7 +38,7 @@ namespace l1t {
       std::vector<unsigned> uparams_;
       std::vector<int> iparams_;
       std::vector<std::string> sparams_;
-      Node(){ type_="unspecified"; version_=0; }
+      Node(){ type_="unspecified"; version_=1; }
```

Comment:
This was requested by @mulhearn so that the emulator defaults to using the 'new' sets of parameters, he did not see any reason for emu to make version=0 parameters.